src/backend/routes/metadata.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  var dtrFhirVersion = "4.0.1";
+  if (process.env.FHIR_VERSION) {
+    dtrFhirVersion = process.env.FHIR_VERSION;
+  }
+
+  var dtrRegisterURL = "http://localhost:3005/register";
```

Comment:
since this is used twice, once here and once in the well-known endpoint, it should be taken from a config file so we only have to modify it in one place.

Reply:
I don't see a config file for storing defaults/constants. Can you point out to me where to put defaults? Thanks.

Reply:
there's a `config.json` file, but it's relatively new.  You'll have to rebase/merge to update this branch to get the config file.

Reply:
Will remove the register URL since it is no longer needed.